contents/handbook/growth/developer-relations.mdx

```diff
@@ -0,0 +1,232 @@
+---
+title: Developer Relations
+sidebar: Handbook
+showTitle: true
+---
+
+*Last Updated On: 16/03/2021*
+
+As a developer-focused company offering a product that, in most cases, will require at least *some* developer involvement (and sometimes a lot of it!), Developer Relations is an important area to us.
+
+While we are not a dev tool, an appropriate PostHog setup requires that the customer, at a minimum, integrate a PostHog library into their codebase. Additionally, engineers may also step in to:
```

Comment:
I would classify us as a devtool since we're adopted that way, even if we have a lot of UX

Reply:
Hmm I'd be against classifying us as a dev tool. We're not really focused on the engineering process - we're a company helping your Product process, even if we're developer-focused.

Reply:
Calling us a dev tool I believe would be misleading.

Reply:
I disagree strongly here.

We are 100% a devtool in the broader market (VC world, and realistically the world of finding candidates to work for us - both see us this way). We target developers with our tool because we believe that analytics and product should be part of the development proecss (even if they aren't in many companies... that's a bad thing though).

I suspect longer term as we think harder about the dev process, product analytics will get closer to that (ie PRs over graphs, rolling out changes behind feature flags as a check in github, that sort of thing). Plugins are a step in this direction too.

The only reason we don't really feel like this is that we started as "an open source alternative to <product-manager-focussed-thing>"

It's kind of semantics, which I think is why we're disagreeing, but I see only downside to not thinking of us as a devtool from the purpose of this piece. We will need people that can code and want to work with developers to join.

Reply:
Historically, product analytics was not position at developers. We're flipping that, PostHog is positioned at developers to help them build more successful products.

Reply:
adjusted this